### ISSUE TITLE
engagement report from first activity #2

### ISSUE THREAD

**Issue by govvin** [OSMPH/events#8]
Connected to #2 .

PDRF and HSBC is requesting for a summary report from the engagement last September.

**Comment by maning** [comment]
I don't think we should do this, it was not part of the initial agreement with PDRF.
Unless you have a diary we can just reference.

**Comment by govvin** [comment]
What were the agreements @maning ? Wala akong diary entry for that activity eh.

**Comment by maning** [comment]
We just run the event no commitments on any writeup.

**Comment by govvin** [comment]
I would've written one, but I just don't have the time right now. :/ Anyway, @ralleon , if you're willing, maybe you could write a one-page brief? You were the designated ringmaster, anyway - top hat, mustache, and megaphone. :laughing:

**Comment by seav** [comment]
I think we should do this. We should document our events and partnerships, even if for ourselves anyway. Did PDRF specify any requirements or can we just do a freeform report on our own?

**Comment by govvin** [comment]
They didn't specify any format, so a free-form document would be just as acceptable.

Off-hand, what I would personally like see in such a brief:
- basic metrics, before and after (if it was a data mapathon)
- topics covered
- significant concerns encountered (by facilitators, by participants), if any
- participants' profile (hoping to see inclusiveness, diversity)
- are participants still editing X period after the engagement?
- feedback from participants

**Comment by maning** [comment]
Yes, for our own purpose we should write something for every event preferrably as an OSM diary.  Its just that for the previous PDRF-HSBC mapathon, we don't have an agreement to do this as part of the engagement.  So the burden is not on us nor this is should be a pre-requisite for the next mapping event we are discussing in #2.

:+1: on the basic outline :point_up:

**Comment by govvin** [comment]
:+1:  for the diary entry, then post links to other social media channels.

Clarification: when Rheeza (?) from HSBC inquired for a report, I replied that I don't think we were required to do any, but she asked if it's possible to get some kind of report, and I thought, the after-event diary entry content would suffice. Since @ralleon coordinated closely with HSBC (and PDRF), I thought it best to ping him directly, and made the suggestion to use the previous diary entries a guide.

From my POV, the "report" is beneficial for everyone - us, the OSM community-at-large, and our other collaborators. The only issue right now is finding someone with some time to gather the details, and write a diary/brief/report.

For future events, I agree that we should make them part of the post-event checklist.
